### PR TITLE
List prohibited headers that are immutable 

### PR DESCRIPTION
**Pull request by tristanlee85**

### CODE REVIEW

**Comment by tristanlee85** [comment]
@ierceg can you confirm this information is accurate? I pulled it from the monorepo source.

File: src/pages/guides/limits.md [hunk 1]
```diff
@@ -35,6 +35,23 @@ This guide describes caveats and limits of {{ PRODUCT_NAME }} platform as applie
...
+
+The following is a list of headers that cannot be modified by your project code. These values are immutable and can only be set by the {{ PRODUCT }} platform.
+
+* `{{ HEADER_PREFIX }}-platform`
+* `{{ HEADER_PREFIX }}-version` 
+* `{{ HEADER_PREFIX }}-t` 
+* `{{ HEADER_PREFIX }}-components`
+* `{{ HEADER_PREFIX }}-status`
+* `{{ HEADER_PREFIX }}-lambda-name`  
+* `{{ HEADER_PREFIX }}-lambda-schedwule-timeout`
```
Comment (ierceg) [inline src/pages/guides/limits.md:48]:
```suggestion
* `{{ HEADER_PREFIX }}-lambda-schedule-timeout`
```

Comment (ierceg) [reply src/pages/guides/limits.md:48]: I would not mention lambda and `received-at` headers. If they get an error, they got an error, but I don't want to start documenting internal headers that are sensitive to people messing with them.

File: src/pages/guides/limits.md [hunk 2]
```diff
@@ -35,6 +35,23 @@ This guide describes caveats and limits of {{ PRODUCT_NAME }} platform as applie
...
+* `{{ HEADER_PREFIX }}-t` 
+* `{{ HEADER_PREFIX }}-components`
+* `{{ HEADER_PREFIX }}-status`
+* `{{ HEADER_PREFIX }}-lambda-name`  
+* `{{ HEADER_PREFIX }}-lambda-schedule-timeout`
+* `{{ HEADER_PREFIX }}-received-at`
+* `host`
+* `x-request-id`
+* `content-length`
+* `via`
```
Comment (tristanlee85) [inline src/pages/guides/limits.md:53]:
```suggestion
* `{{ HEADER_PREFIX }}-platform`
* `{{ HEADER_PREFIX }}-version` 
* `{{ HEADER_PREFIX }}-t` 
* `{{ HEADER_PREFIX }}-components`
* `{{ HEADER_PREFIX }}-status`
* `host`
* `x-request-id`
* `content-length`
* `via`
```

**Review by ierceg — APPROVED** [review]

**Pull request merged**